rtlib/include/rt/DisegniSegmenter.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+// C++ Standard Libraries
+#include <string>
+#include <utility>
+#include <vector>
+
+// External Libraries
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/core/utility.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/opencv.hpp>
+#include <stdio.h>
+// Image Blur Library
+#include <opencv2/highgui/highgui_c.h>
+#include <opencv2/imgproc/imgproc.hpp>
```

Comment:
This header only makes use of `vector` and `cv::Mat`, so remove everything except for:

```c++
#include <vector>

#include <opencv2/core.hpp>
```

Your class cpp only needs:
```c++
#include "rt/DisegniSegmenter.hpp"

#include <limits>

#include <opencv2/imgproc.hpp>
```

Your application cpp only needs:
```c++
#include <iostream>

#include <opencv2/core.hpp>
#include <opencv2/highgui.hpp>
#include <opencv2/imgcodecs.hpp>
```